draft-ietf-scitt-architecture.md

```diff
@@ -1007,7 +1007,8 @@ This enables the gradual transition to stronger algorithms, including e.g. post-
 ### Transparency Service Client Applications
 
 Authentication of Client applications is out of scope for this document.
-Transparency Services MUST authenticate both Client applications and the Issuer of Signed Statements in order to ensure that implementation of specific authentication and authorization policies are enforced.
+Transparency Services MAY authenticate Client applications.
+Transparency Services MUST authenticate the Issuer of Signed Statements in order to ensure that implementation of specific authentication and authorization policies are enforced.
```

Comment:
Authenticating issuers is different than authenticating signed statements.

I think this sentence is supposed to say:

TS MUST check signature on signed statements.

Or with including authenticating the issuer:

TS MUST check signature on signed statements and ensure the public key that verifies the signature is trusted by the TS to identify the issuer.